testng-core/src/main/java/org/testng/internal/DataProviderLoader.java

```diff
@@ -0,0 +1,43 @@
+package org.testng.internal;
+
+import java.io.IOException;
+import java.io.InputStream;
+import org.testng.log4testng.Logger;
+
+public class DataProviderLoader extends ClassLoader {
+  private static final int BUFFER_SIZE = 1 << 20;
+  private static final Logger log = Logger.getLogger(DataProviderLoader.class);
```

Comment:
both these static data members can be removed as they are no longer being used.

Reply:
removed